dragon-scrolls/level-1/readme.md

```diff
@@ -5,13 +5,54 @@ of Object-Oriented programming terminology, basic Python data structures,
 and setting up PyCharm for your projects.
 
 ## Class Setup
-Prior to each class, there are a few things to do:
```

Comment:
Can you figure out how to revert this change?  I don't want this version!  You've deleted a line from the dragon-scroll!